src/test/java/com/mentalhealthapi/mentalhealthapi/MentalHealthApiApplicationTests.java

```diff
@@ -41,6 +41,12 @@ private void whenSearchDisorderWithNameDepression() {
     }
 
     private void ReturnsDepressionForNameDepression() {
+        Disorder disorder = new Disorder();
+        disorder.setId(0);
+        disorder.setName("Depression");
+        disorder.setDescription("Disorder 0");
+        disorder.setPageNumber("20");
+        disorder.setPopulationPercent((float) 0.7);
```

Comment:
A unit test failed and the part that was missing was an object with correct attributes. The change should be made based on the "Code Review in GitHub, IntelliJ Idea" video that goes with the Code Review assignment, where it was stated that all unit tests should pass.

Reply:
I fixed this bug shortly after a member pushed it to master.